cgmanifest.json

```diff
@@ -189,6 +189,15 @@
             }
          }
       },
+      {
+         "component":{
+            "type":"git",
+            "git":{
+               "commitHash":"3ad2bd19d6f0a2475805e9aa5ae8734a0d60fbaa",
+               "repositoryUrl":"https://github.com/onnx/onnx-tensorrt"
```

Comment:
I guess this is conflicting with steven's update?

Reply:
removed my edits in this file.